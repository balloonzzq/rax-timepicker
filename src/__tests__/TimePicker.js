import {createElement, Component} from 'rax';
import renderer from 'rax-test-renderer';
import TimePicker from '..';

describe('TimePicker', () => {
  it('should render TimePicker', () => {
    const tree = renderer.create(
      <TimePicker selectedValue={'10:10'} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

