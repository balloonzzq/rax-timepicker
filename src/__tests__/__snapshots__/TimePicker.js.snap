// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimePicker should render TimePicker 1`] = `
Object {
  "attributes": Object {
    "type": "time",
    "value": "10:10",
  },
  "eventListeners": Object {
    "change": [Function],
  },
  "tagName": "INPUT",
}
`;
